index.bs

```diff
@@ -806,6 +806,12 @@ An attribution source is a [=struct=] with the following items:
 :: A [=trigger-data matching mode=].
 : <dfn>debug cookie set</dfn> (default false)
 :: A [=boolean=].
+: <dfn>attribution scope limit</dfn>
+:: Null or a positive 32-bit integer, number of distinct [=attribution source/attribution scopes=] allowed per [=attribution source/source origin=].
```

Comment:
Should this be per reporting origin?

Reply:
Isn't it per reporting origin x destination?

Reply:
I think I mixed source origin and report origin somehow. Fixed this.